ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -941,6 +941,357 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-advanced
+- as: aws-rosa-sts-advanced-f360-content
+  cron: 9 21 2,9,16,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_SECURITY_GROUP: "true"
+      CHANNEL_GROUP: nightly
+    test:
+    - as: test-e8-rhcos4
+      cli: latest
+      commands: |
+        set -x
+        export PROFILE=e8
```

Comment:
Another thing we can try do to would be to just run PCI-DSS and CIS profiles against ROSA initially. We probably don't need to run all profiles at first.

- ocp4-cis
- ocp4-cis-node
- ocp4-pci-dss
- ocp4-pci-dss-node